src/app/qgsvectorlayersavestyledialog.cpp

```diff
@@ -156,6 +176,55 @@ void QgsVectorLayerSaveStyleDialog::readUiFileContent( const QString &filePath )
   }
 }
 
+void QgsVectorLayerSaveStyleDialog::setupMultipleStyles()
+{
+  // Show/hide part of the UI according to multiple style support
+  if ( ! mSaveOnlyCurrentStyle )
+  {
+    const auto mgr { mLayer->styleManager() };
+    const auto constStyles = mgr->styles();
```

Comment:
auto :-1:

Reply:
Could we try to come up with a coding guideline for auto? As was suggested a few times already, we should limit auto to the for loops iterating through lists / maps really.

Declaring a variable type goes right along using informative/clear variable names.

Reply:
I know about the policy: this is the first iteration and I'll clean it up before merge. I was more interested in comments regarding the functionality in this phase. 
But thank you for reminding me.